<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRADMainWindow</class>
 <widget class="QMainWindow" name="XRADMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="consoles">
      <item>
       <widget class="QLabel" name="stdout_title">
        <property name="text">
         <string>stdout console</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="XRAD_GUI::stdio_console" name="stdout_console">
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stderr_title">
        <property name="text">
         <string>stderr console</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="XRAD_GUI::stdio_console" name="stderr_console"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>Functions</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <widget class="QMenu" name="menuPriority">
     <property name="title">
      <string>Priority</string>
     </property>
     <addaction name="actionPriorityDefault"/>
     <addaction name="actionPriorityNormal"/>
     <addaction name="actionPriorityIdle"/>
    </widget>
    <addaction name="actionQuit"/>
    <addaction name="actionDebug"/>
    <addaction name="menuPriority"/>
   </widget>
   <widget class="QMenu" name="menuLogs">
    <property name="title">
     <string>Logs</string>
    </property>
    <addaction name="actionLogsShowSpecialCharacters"/>
    <addaction name="actionLogsWriteToFiles"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionLangRu"/>
     <addaction name="actionLangEn"/>
    </widget>
    <addaction name="actionAllowStayOnTop"/>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuFunction"/>
   <addaction name="menuProgram"/>
   <addaction name="menuLogs"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLogsWriteToFiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Write logs to files</string>
   </property>
  </action>
  <action name="actionLogsShowSpecialCharacters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show special characters</string>
   </property>
  </action>
  <action name="actionAllowStayOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Allow stay on top</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionPriorityDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionPriorityNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionPriorityIdle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Idle</string>
   </property>
  </action>
  <action name="actionLangRu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionLangEn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XRAD_GUI::stdio_console</class>
   <extends>QPlainTextEdit</extends>
   <header>stdio_console.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
